<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregar_tarjeta</class>
 <widget class="QDialog" name="agregar_tarjeta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Formulario de tarjeta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de tarjeta </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de tarjeta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PIN de tarjeta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Monto de tarjeta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del cliente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID cliente</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tipo_tarjeta_nueva">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Debito</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Credito Platino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Credito Oro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Credito Clasica</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="numero_tarjeta_nueva">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pin_tarjeta_nueva">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="monto_tarjeta_nueva">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre_cliente_nuevo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_cliente_nuevo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../iconos/check-64.png</normaloff>../iconos/check-64.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../iconos/credit-card.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../iconos/credit-card.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
